src/twisted/web/server.py

```diff
@@ -453,6 +454,14 @@ def getSession(self, sessionInterface=None, forceNotSecure=False):
 
         session = getattr(self, sessionAttribute)
 
+        if session:
```

Comment:
This is probably better written as `if session is not None:` (and the existing line below better written as `if session is None:`